unleash-yggdrasil/src/lib.rs

```diff
@@ -529,11 +529,12 @@ impl EngineState {
         context: &Context,
         external_values: &Option<HashMap<String, bool>>,
     ) -> Option<VariantDef> {
-        self.get_toggle(name).and_then(|toggle| {
+        self.get_toggle(name).map(|toggle| {
```

Comment:
This means that we now return the disabled variant if the feature exists but no variants are defined. This was previously not handled and was causing a spec test to fail (thank you client spec)